pkg/constants/constants.go

```diff
@@ -51,6 +51,7 @@ const (
 	CLUSTER_RESOURCES_ROLE_BINDINGS               = "rolebindings"
 	CLUSTER_RESOURCES_CLUSTER_ROLES               = "clusterroles"
 	CLUSTER_RESOURCES_CLUSTER_ROLE_BINDINGS       = "clusterRoleBindings"
+	CLUSTER_RESOURCES_PRIORITY_CLASS              = "priority-classes"
```

Comment:
Can we change this to `priorityclasses` to conform with the other resource names?

Reply:
It maps to `priorityclasses.scheduling.k8s.io`

Reply:
I was following naming that already exists for storage classes, thats the reason for the dash. I'll update the PR.

```
	CLUSTER_RESOURCES_STORAGE_CLASS               = "storage-classes"
```